src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -39,7 +40,7 @@ public class GraphQLHttpMiddleware<TSchema> : GraphQLHttpMiddleware
         IDocumentExecuter<TSchema> documentExecuter,
         IServiceScopeFactory serviceScopeFactory,
         GraphQLHttpMiddlewareOptions options)
-        : base(next, serializer, options)
+        : base(next, serializer, options, Array.Empty<IWebSocketHandler>())
```

Comment:
The first thought looking at this signature - why not typed handler - `IWebSocketHandler<TSchema>`?

Reply:
The “base” class uses untyped IDocumentExecuter and untyped IWebSocketHandler. This ctor isn’t finished because the supporting class isn’t done yet.